data_set/taskset/sanity_check.md

```diff
@@ -0,0 +1,276 @@
+
+# Details
+For each sampled post, we try the original prompt as well as an adaptation that's closer on how Tellina will be used during the user experiment.
+
+For each prompt, we report the top 3 suggestions for traceability.
+
+## 1. http://www.bashoneliners.com/oneliners/70/ (Print the lines of file2 that are missing in file1)
+One-liner: `comm -23 file2 file1`
+
+### Prompt: Print the lines of file2 that are missing in file1
+- `comm -1 -3 file1 file2`
+- `sort file1 file2 | uniq`
+- `join < ( sort file2 )`
+
+Helpful
+
+### Adapted prompt: Print the lines in foo.txt that do are not in bar.txt
+- `comm -1 -3 foo.txt bar.txt`
+- `comm -1 -3 foo.txt foo.txt`
+- `join -v 1 foo.txt`
+
+Helpful
+
+## 2. https://www.commandlinefu.com/commands/view/42/display-which-distro-is-installed (Prompt Display which distro is installed) 
+One-liner: `cat <file>`
+
+### Prompt: Display which distro is installed
+- `find . -name '[__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK][__SP__UNK]__SP__UNK'
+
+Unhelpful
+
+### Adapted prompt: Display the content of index.html
+- `cat index.html`
+- `rev index.html`
+- `find . -type f -exec cat {} \; -exec cat {} \; -exec cat {} \; -exec cat {} \; -exec cat {} \;`
+
+Helpful
+
+## 3. https://www.commandlinefu.com/commands/view/1295/quickly-rename-a-file (Quickly rename a file)
+One-liner: `mv filename.{old,new}`
+
+### Prompt: Quickly rename a file
+- `find . -type f -exec ls -s {} \; | xargs -0 -I {} rm {}`
+- `find . -type f -exec ls -s {} \; | xargs -0 -I {} rename 's/ /_/g' {}`
+- `find . -type f -exec ls -s {} \; | sort -n | head -n quickly`
+
+Unhelpful
+
+### Adapted prompt: Rename index.txt to index.csv
+- `mv index.txt index.csv`
+- `rename 's/\.txt/index.txt' index.csv`
+- `rename 's/\.txt/.txt/' index.csv`
+
+Helpful
+
+## 4. https://stackoverflow.com/questions/1529946/ (Linux: copy and create destination dir if it does not exist)
+One-liner: `mkdir -p "$d" && cp file "$d"`
+
+### Prompt: Copy and create destination dir if it does not exist
+- `cp -n`
+- `cp -r dir_dir dir1`
+- `cp -R dir /path/to/dir`
+
+Helpful (Copy directory)
+
+### Adapted prompt: Copy folder src in new folder a/b
+- `find a/b -type d -exec mkdir -p {} \;`
+- `find a/b -type d -print0 | xargs -0 -I {} cp -t a/b {}`
+- `find a/b -type d -print0 | xargs -0 -I {} cp a/b {}`
+
+Helpful (`mkdir -p`)
+
+## 5. https://stackoverflow.com/questions/3137094/
+One-liner: `wc -l <filename>`
+How to count lines in a document?
+
+### Prompt: How to count lines in a document?
+- `cat /dev/urandom | tr -d -c '[0-9]+' | wc -c`
+- `cat /dev/urandom | tr -d -c '[0-9] ' | wc -c`
+- `cat /dev/urandom | tr -d -c '[a-z]' | wc -c`
+
+Unhelpful
+
+### Adapted prompt: Count lines in index.html
+- `wc -l index.html`
+- `cat index.html | wc -l`
+- `wc index.html`
+
+Helpful
+
+## 6. https://superuser.com/questions/482435/
```

Comment:
Can you put the prompt on this line as well for consistency.

Reply:
For sure! IDK why it's not there.